draft-ietf-dprive-dnsoquic.md

```diff
@@ -853,7 +882,7 @@ conducted by Stephane Bortzmeyer helped improve the definition of the protocol.
 
```

Comment:
SHould we add a case where the server receives a Quic reset stream request?

Reply:
What section are you proposing adding this to? I presume there is a case where a client might want to send this if a server appears to take no action after a STOP_SENDING?

Reply:
No. If the client has sent a full request, the client's side of the stream is closed. Resetting a stream after it is closed has no effect. The only potential actions after sending STOP_SENDING are to wait for the reset, or if it does not come continue reading some data and maybe discard it, or if that's too much data close the connection.

The client reset scenario would be:
* The server receives a part of a request, say the first bytes, but has not received the full request yet.
* The server at that point receives a reset stream request, meaning the client has dropped the stream and will never send the missing bytes to complete the request.
That is a very corner case scenario, because most requests will be sent in one shot. It smells like the client is trying to play tricks with the server. Maybe it should be mentioned in the security section. Or maybe we should just keep that in mind for a later release.

Reply:
I'm not sure we need to add it to this version... servers will have to be robust to other cases like never getting a full query with no stream reset so I feel like this falls into the general error case, which we can clarify later if needed.

Reply:
Ok, we can wait